### PR TITLE
Fix: Versioning display

### PR DESCRIPTION
**Pull request by silvav**
fixes bmitch/churn-php#211 

churn -V
churn --version

### CODE REVIEW

File: tests/Unit/ChurnPHPApplicationTest.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+namespace Churn\Tests\Unit;
+
+use Churn\ChurnPHPApplication;
+use Churn\Tests\BaseTestCase;
+
+class ChurnPHPApplicationTest extends BaseTestCase
+{
+    const SEMANTIC_VERSIONING_PATTERN = '\d+\.\d+.\d+';
+
+    /** @test **/
```
Comment (villfa) [inline tests/Unit/ChurnPHPApplicationTest.php:12]: I'm nitpicking here but it should be:
```
    /** @test */
```

See: https://docs.phpdoc.org/latest/guides/docblocks.html

Comment (silvav) [reply tests/Unit/ChurnPHPApplicationTest.php]: Thank you for point it out!

I've applied the fix through the test folder, please see c9b1ef3.

**Comment by villfa** [comment]
I'm closing this PR since I've chosen another approach in #225.
Thank you @silvav for your work. I've cherry-picked one of your commits.

**Comment by silvav** [comment]
> I'm closing this PR since I've chosen another approach in #225.
> Thank you @silvav for your work. I've cherry-picked one of your commits.

I am glad this issue is solved, thanks for that.

When a new dependency is introduced, the stability of this package is reduced.
`Stability = (number of outgoing dependencies) / (number of total incoming and outgoing dependencies)`

https://drive.google.com/file/d/0BwhCYaYDn8EgZjI3OTU4ZTAtYmM4Mi00MWMyLTgxN2YtMzk5YTY1NTViNTBh/view

**Pull request closed without merging**